code/datums/uplink_item.dm

```diff
@@ -262,7 +262,7 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/stealthy_weapons/detomatix
 	name = "Detomatix PDA Cartridge"
-	desc = "When inserted into a Personal Data Assistant, this cartridge gives you four opportunities to detonate PDAs of crewmembers who have their message feature enabled. The concussive effect from the explosion will knock the recipient out for a short period, and deafen them for longer. It has a chance to detonate your PDA."
+	desc = "When inserted into a Personal Data Assistant, this cartridge gives you five opportunities to detonate PDAs of crewmembers who have their message feature enabled. The concussive effect from the explosion will knock the recipient out for a short period, and deafen them for longer. It has a chance to detonate your PDA."
```

Comment:
WOW UNATOMIC CHANGE UGH

Reply:
I genuinely have no idea what that is, can someone check if it's five right now ?

Reply:
>Look at dylan, this lad is bad enough to revert bug fixes #21598

Reply:
Gotcha, autorebase failure. Lemme push up a fix